README.md

```diff
@@ -173,6 +173,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [Gzip](https://crystal-lang.org/api/Gzip.html) - readers and writers of gzip format (Crystal stdlib)
  * [INI](https://crystal-lang.org/api/INI.html) - INI file parser (Crystal stdlib)
  * [JSON](https://crystal-lang.org/api/JSON.html) - parsing and generating JSON documents (Crystal stdlib)
+ * [JSON tools](https://github.com/impatienttraveller/json-tools) - an implementation of JSON Patch and Pointer RFC's
```

Comment:
```suggestion
 * [JSON tools](https://github.com/impatienttraveller/json-tools) - An implementation of JSON Patch and Pointer RFC's
```

Reply:
There are mixed cases throughout the list, maybe someone could send a PR fixing it.